### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by warabe**
1. Asking for feedback
2. Page template for the privacy policy
3. Option to skip chatops altogether
4. Link to a "bring your own bot" page
5. Correct link for the "Open StackStorm" button
6. Extended clean-up: command chain instead of a single command.

### CODE REVIEW

File: st2installer/templates/progress.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
       <li><a href="http://stackstorm.com/blog" target="_blank">Blog</a></li>
     </ul>
     <div id="next-button">
-      <a href="http://google.com" id="step-next">Open StackStorm</a>
+      <a href="https://${hostname}" id="step-next">Open StackStorm</a>
```
Comment (jfryman) [inline st2installer/templates/progress.html]: Does not go to the right place. Went to https:localhost/

failed_to_open_page

**Comment by jfryman** [comment]
The behavior of `byob` is a bit odd. I have to commit to install first, so that the hostname is saved in some way, and then the page will render. I found myself clicking the `byob` link first, so `<hostname>` is what ended up rendered.

**Comment by jfryman** [comment]
I like the additions. Small fixes. I also added a few fixes to get things working, and totally screwed the branch. Sorry. :\

Also, final notes on what to do with the 'permission' bit:
* create a new key in the `workroom.yaml` that reads as follows:
```
st2::kvs:
  st2::install_uuid: (randomly generated UUID)
  st2::collect_anonymous_data: (value of user input)
```

**Comment by jfryman** [comment]
Also, the privacy policy endpoint is not needed. We are going to maintain the privacy policy upstream. https://github.com/StackStorm/st2installer/commit/5362d8f37350847ed0de29d324618d1c2d9570e7

**Comment by jfryman** [comment]
I'm merging this, as the PR is getting pretty big. Most of the things work, so I'm going to converge and we can :hammer: out :bug: as necessary.

**Comment by jfryman** [comment]
Ok, so master now has what I need. How does this make it back to 'stable'?

**Comment by warabe** [comment]
I'll make a new PR in a minute.

**Pull request merged**